db/pf-schema-X.Y.Z.sql

```diff
@@ -1299,10 +1299,50 @@ CREATE TABLE auth_log (
   KEY  attempted_at (attempted_at)
 ) ENGINE=InnoDB;
 
+DELIMITER //
+CREATE TRIGGER `log_event_auth_log_insert` AFTER INSERT ON `auth_log`
```

Comment:
How about:

- RADIUS audit log?
- DNS audit log?
- DHCP option 82?

Reply:
The radius audit log has lots of fields. Which one should be logged?

Reply:
Propose something